### PR TITLE
feat(explorer): truncated orderId with CSS

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

~Shorten orderId on explorer like an address~
Truncate orderId with CSS

# To Test

1. Open explorer
2. Search for an order id
* OrderId should start with `0x`, have an elipsis in the ~middle and~ end ~with the last 4 chars of the full id~.

3. Copy the full orderId using the copy button
4. Search in the page (ctrl+f) for the full order id
* It should highlight both the header and the table order ids, even though the full text is not displayed
image

5. Open an address view
* Order ids should be in the format described above
image

6. Open the transaction details view
* Order ids should be in the format described above
image

### CODE REVIEW

**Review by shoom3301 — APPROVED** [review]

**Review by anxolin — APPROVED** [review]

**Review by elena-zh — COMMENTED** [review]
Hey @alfetopito , great job! One nitpick: I don't like how the orderID  is displayed on a [TX details page:](https://explorer-dev-git-fix-explorer-shortid-cowswap.vercel.app/gc/tx/0x3400255ac104b94b237d2469f5f6de6211f468fcd0c54f4fcc5bf57a40040d83?tab=orders) 
image
Could you please fix it?

**Comment by elena-zh** [comment]
Hey @alfetopito , still there are some issues with orderID displaying on a reduced screen: 
image

**Review by elena-zh — APPROVED** [review]
Looks good now, thank you!

File: apps/explorer/src/components/common/TruncatedText/index.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  display: block;
+`
+
+type TruncatedProps = {
+  text: string
+  width?: string
+  className?: string
+}
+
+export function TruncatedText(props: TruncatedProps) {
```
Comment (shoom3301) [inline apps/explorer/src/components/common/TruncatedText/index.tsx:17]: Would be awesome to move the component into `libs/ui`

**Review by shoom3301 — APPROVED** [review]

**Pull request merged**